src/trio/_tests/test_subprocess.py

```diff
@@ -339,15 +338,30 @@ async def test_run() -> None:
     # invalid combinations
     with pytest.raises(UnicodeError):
         await run_process(CAT, stdin="oh no, it's text")
-    with pytest.raises(ValueError):
+    with pytest.raises(
+        ValueError,
+        match=r"^stdout=subprocess\.PIPE is only valid with nursery\.start, since that's the only way to access the pipe(; use nursery\.start or pass the data you want to write directly)*$",
```

Comment:
this string also makes sense to save in a variable and reuse

Reply:
Changed in f89cd0a